cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - name: IMAGE
           value: $(params.IMAGE_NAME)
         - name: VERSION
-          value: "8"
+          value: "12-ubi8"
```

Comment:
why `12-ubi8` when pkg/reconciler/openshift/tektonaddon/pipelinetemplates/pipelinetemplates.go defines `14-ubi8`?

Reply:
This I missed as it in the samples. Updated now